src/main/scala/org/scalajs/tools/tsimporter/parser/TSDefParser.scala

```diff
@@ -47,7 +47,7 @@ class TSDefParser extends StdTokenParsers with ImplicitConversions {
       "{", "}", "(", ")", "[", "]", "<", ">",
       ".", ";", ",", "?", ":", "=", "|",
       // TypeScript-specific
-      "...", "=>"
+      "...", "=>", "&"
```

Comment:
`&` should be next to `|` above, ideally.

Reply:
Done in https://github.com/sjrd/scala-js-ts-importer/pull/46/commits/0922a5431ae5f9d72d3ef5157a2b6afcaeb6cd6a